doc/client_api.md

```diff
@@ -294,7 +294,7 @@ It updates the audio and video maximum bandwidth for a publisher or a subscriber
 It can also be used in remote streams to toggle `slideShowMode`
 
 <example>
```

Comment:
Maybe not as part of this PR but we should update this section since we can now change many parameters dynamically (mute, slideshow, simulcast options, etc.)